README.md

```diff
@@ -1,22 +1,25 @@
 # Nva Datacite MDS
```

Comment:
What is really MDS?

Reply:
Abbreviation of Metadata Store I think (DataCite language)